modules/rosa-sharing-vpc-dns-and-roles.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-dns-and-roles_{context}"]
+= Creating your DNS and cluster creation roles
+
+After the VPC-owning AWS account user creates a virtual private cloud, subnets, and an IAM role for sharing the VPC resources, the cluster-creating AWS account must create a DNS domain and operator roles to share with the VPC-owning AWS account.
```

Comment:
I think you need the word "user" after "cluster-creating AWS account". "Operator" should also be capitalized.